src/Components/Blueprints/DeleteBlueprintModal.tsx

```diff
@@ -7,25 +7,27 @@ import {
   ModalVariant,
 } from '@patternfly/react-core';
 
+import {
+  selectBlueprintSearchInput,
+  selectSelectedBlueprint,
+} from '../../store/BlueprintSlice';
+import { useAppSelector } from '../../store/hooks';
 import { useGetBlueprintsQuery } from '../../store/imageBuilderApi';
 
 interface DeleteBlueprintModalProps {
   onDelete: () => Promise<void>;
-  selectedBlueprint: string | undefined;
   isOpen: boolean;
   onClose: () => void;
 }
 
 export const DeleteBlueprintModal: React.FunctionComponent<
   DeleteBlueprintModalProps
-> = ({
-  onDelete,
-  selectedBlueprint,
-  isOpen,
-  onClose,
-}: DeleteBlueprintModalProps) => {
+> = ({ onDelete, isOpen, onClose }: DeleteBlueprintModalProps) => {
+  const selectedBlueprint = useAppSelector(selectSelectedBlueprint);
+  const blueprintsSearchQuery = useAppSelector(selectBlueprintSearchInput);
```

Comment:
Would you mind changing this to `blueprintSearchInput`? I find it easier to understand the code when the name of the variable in the React component is the same as it is in the store.